src/commands/info/help.ts

```diff
@@ -75,23 +84,23 @@ export default class extends Command {
 
 			const splitStr = Util.splitMessage(helpStr, { char: '\n' });
 
-			let notified = false;
+			let part = 1;
 			splitStr.forEach(helpMsg => {
-				msg.author.send(helpMsg)
-					.then(() => {
-						if (!notified) {
-							if (msg.channel.type !== 'DM') msg.channel.send('I\'ve sent all commands to your DMs');
-							notified = true;
-						}
-					})
-					.catch(() => {
-						if (!notified) {
-							msg.channel.send('I couldn\'t send you a DM. Please enable DMs and try again');
-							notified = true;
-						}
-					});
+				const embed = new MessageEmbed()
+					.setTitle(`Commands Part ${part}`)
+					.setDescription(helpMsg)
+					.setColor('RANDOM');
+				interaction.user.send({ embeds: [embed] });
```

Comment:
Doubling down on this it would be best to make the response ephemeral (in the server) instead of a series of DM messages so that this problem is entirely avoided.